presto-native-execution/presto_cpp/main/operators/tests/UnsafeRowShuffleTest.cpp

```diff
@@ -1177,6 +1177,6 @@ TEST_F(UnsafeRowShuffleTest, shuffleInterfaceRegistration) {
 
 int main(int argc, char** argv) {
   testing::InitGoogleTest(&argc, argv);
-  folly::init(&argc, &argv, false);
```

Comment:
It would be nice to extract these changes into a separate PR. I believe they do not require the upgrades. This would help keep this PR small and focused.

Reply:
These are related to the folly upgrade that is part of the Velox upgrade here.